Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -5,7 +5,7 @@ import CardContent from '../CardComponents/CardContent.js';
 
 const CardContainer = () => {
         return (
-        <a href="https://reactjs.org/">
+        <a href="https://reactjs.org/"  target="_blank">
```

Comment:
I like this idea of wrapping the entire container in a link tag. Nice way to achieve the goal without bringing click handlers into the mix.